### ISSUE TITLE
Keyboard focus indicator is not visible on the 'Next' and 'Previous' control.

### ISSUE THREAD

**Issue by 232amansingh** [microsoft/BotFramework-WebChat#5744]
### Is it an issue related to Adaptive Cards?

Yes, this is related to Adaptive Card. However, it does not repro on Adaptive Cards Designer.

### What is the PWD impact?

When the keyboard focus indicator is not visible on the “Next” and “Previous” controls, keyboard users cannot determine which element currently has focus. This makes navigation unpredictable and prevents users from reliably moving through paginated or carousel content.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Repro Steps: ​           
1. Open URL: Web Chat in edge browser.
2. Ask query 'Carousel'.
3. Navigate to next and previous control.
4. Observe the issue that Keyboard focus indicator is not visible on the 'next' and 'previous' control. 

### What do you expect?

Keyboard focus indicator should be visible on the 'next' and 'previous' control. 

### What actually happened?

Keyboard focus indicator is not visible on the 'next' and 'previous' control. 

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/67f1116a-5141-4509-8833-b9a4211db1f1

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/focus-visible

### Adaptive Card JSON

```json

```
